raiden/storage/restore.py

```diff
@@ -210,9 +210,12 @@ def get_event_with_balance_proof_by_locksroot(
 def get_state_change_with_transfer_by_secrethash(
     storage: SerializedSQLiteStorage, secrethash: SecretHash
 ) -> Optional[StateChangeRecord]:
-    filters = [{"transfer.lock.secrethash": to_hex(secrethash)}]
+    filters = [
+        {"from_transfer.lock.secrethash": to_hex(secrethash)},
+        {"transfer.lock.secrethash": to_hex(secrethash)},
+    ]
```

Comment:
I don't yet understand why we need this change. How can I get an overview of how different state changes look?

Reply:
We want to detect both `ActionInitMediator` (which has a `from_transfer` field), and `ActionInitTarget` (which has a `transfer` field) here.

https://github.com/raiden-network/raiden/blob/8fc24e7663a99b4e99917ef21a46df7c51e8732e/raiden/transfer/mediated_transfer/state_change.py#L41

Reply:
Makes sense. Thanks for the pointer.